src/write.ts

```diff
@@ -106,14 +106,15 @@ function findAlerts(curSuite: Benchmark, prevSuite: Benchmark, threshold: number
     return alerts;
 }
 
-function getCurrentRepo() {
-    const repo = github.context.payload.repository;
-    if (!repo) {
-        throw new Error(
-            `Repository information is not available in payload: ${JSON.stringify(github.context.payload, null, 2)}`,
-        );
-    }
-    return repo;
+function getCurrentRepoMetadata() {
+    return {
+        name: github.context.repo.repo,
+        owner: {
+            login: github.context.repo.owner,
+        },
+        // eslint-disable-next-line @typescript-eslint/camelcase
+        html_url: `https://github.com/${github.context.repo.owner}/${github.context.repo.repo}`,
+    };
```

Comment:
I suggest a small readability improvement here :)
```suggestion
    const { repo, owner } = github.context.repo
    return {
        name: repo,
        owner: {
            login: owner,
        },
        // eslint-disable-next-line @typescript-eslint/camelcase
        html_url: `https://github.com/${owner}/${repo}`,
    };
```

Reply:
I made these changes!